Go 1.15: conversion from int to string yields a string of one rune, not a string of digits

Go 1.15 rc 1 on Fedora Rawhide
```
Testing    in: /builddir/build/BUILD/go-diff-1.1.0/_build/src
         PATH: /builddir/build/BUILD/go-diff-1.1.0/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/go-diff-1.1.0/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/sergi/go-diff/version=1.1.0 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/sergi/go-diff
github.com/sergi/go-diff/diffmatchpatch
# github.com/sergi/go-diff/diffmatchpatch
./patch.go:327:18: conversion from int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
FAIL	github.com/sergi/go-diff/diffmatchpatch [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/diffmatchpatch/patch.go
+++ b/diffmatchpatch/patch.go
@@ -324,7 +324,7 @@ func (dmp *DiffMatchPatch) PatchAddPadding(patches []Patch) string {
 	paddingLength := dmp.PatchMargin
 	nullPadding := ""
 	for x := 1; x <= paddingLength; x++ {
-		nullPadding += string(x)
+		nullPadding += string(rune(x))
 	}
 
 	// Bump all the patches forward.

